app/backend/approaches/retrievethenread.py

```diff
@@ -63,9 +63,9 @@ def run(self, q: str, overrides: dict[str, Any]) -> Any:
 
         # Use semantic ranker if requested and if retrieval mode is text or hybrid (vectors + text)
         if overrides.get("semantic_ranker") and has_text:
-            r = self.search_client.search(query_text,
+            r = await self.search_client.search(query_text,
                                           filter=filter,
-                                          query_type=QueryType.SEMANTIC,
+                                          query_type=QueryType.SIMPLE,
```

Comment:
Why this change ?

Reply:
Good catch, that was unintentionally committed, thats a change I make locally due to running out of semantic search credits this month. Reverted.